src/pages/education.tsx

```diff
@@ -0,0 +1,218 @@
+import * as React from 'react'
+
+import Page from '../components/Page'
+import Container from '../components/Container'
+import IndexLayout from '../layouts'
+import Logos from '../components/Logos';
+import { colors , breakpoints} from '../styles/variables';
+import { getEmSize } from '../styles/mixins';
+import * as icons from '../resources/icons';
+import styled from '@emotion/styled';
+import PricingBox from '../components/PricingBox';
+
+const Hero = styled.section`
+    display: flex;
+    min-height: 500px;
+    margin-top: 100px;
+    .text-box {
+        width: 50%;
+        @media(max-width: 768px) {
```

Comment:
Please use the same `@media` queries as in the rest of the code, e.g.:

```suggestion
        @media (max-width: ${getEmSize(breakpoints.md - 1)}em) {
```